tests/Node170.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tests;
+
+use PHPUnit\Framework\TestCase;
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+class Node170 extends TestCase
+{
+    private function call($body) {
```

Comment:
This method could be inherited from an abstract test class used for all languages. We can also have all languages tests on the same PHPUnit class with different methods..

Reply:
I see. Functions (input and output) are same, just the implementation is different. Ill make sure to abstract it.

Reply:
👌